browser/flagr-ui/src/components/Flag.vue

```diff
@@ -835,15 +839,29 @@ export default {
     },
     toggleShowMdEditor () {
       this.showMdEditor = !this.showMdEditor
+    },
+    setJsonEdiorModeWhenMounted () {
+      if (!this.$refs) {
+        return
+      }
+
+      const jsonEditorRefs = Object.keys(this.$refs).filter(key => key.startsWith('json-editor'))
+
+      jsonEditorRefs.forEach(ref => {
+        this.$refs[ref][0].editor.setMode('code')
+      })
+
+      clearInterval(this.refsIntervalId)
     }
   },
   mounted () {
     this.fetchFlag()
+    this.refsIntervalId = setInterval(this.setJsonEdiorModeWhenMounted, 50)
   }
 }
 </script>
 
-<style lang="less">
+<style lang="less" scoped>
```

Comment:
We still need the unscoped style to make sure it can be applied to `.el-*` elements. That's why I originally didn't use scoped here.

Reply:
I prefer always to scope my components style and to add global styles at root scope. What do you think?

Reply:
Alright